ext/standard/math.c

```diff
@@ -204,83 +204,6 @@ PHPAPI double _php_math_round(double value, int places, int mode) {
 }
 /* }}} */
 
-/* {{{ php_asinh
-*/
-static double php_asinh(double z)
-{
-#ifdef HAVE_ASINH
```

Comment:
As far as I see, definitions like HAVE_ASINH or HAVE_ACOSH could also be removed if C99 is really allowed.

Reply:
Indeed, I forgot to that.